### DISCUSSION TITLE
Can't stretch components to the max width of the layout

### DISCUSSION THREAD

**Discussion by aviNrich** [Budibase/budibase#14497] [Help] [answered]
Hi,
I am examining BB for my company.
I add a table to the canvas, but I cant make it full width.
you can see in the video that also setting the width from the style window is not working as well.
This happns also when adding an empty container component.
What am I doing wrong?
Thanks

![Untitled Video September 2, 2024 12_57 PM](https://github.com/user-attachments/assets/c25caef0-976a-4b40-9fa3-0571d0b9fa8b)

**Comment by ConorWebb96** [comment]
Hey @aviNrich,

- What version of Budibase are you running?
- Are you self-hosted or cloud?
- What browser are you using?

**Reply by aviNrich** [reply]
2.31.3
docker compose
chrome

Thanks

**Reply by ConorWebb96** [reply, marked as answer]
Could you see if you are getting any console errors? I've just tried on my chrome browser and wasn't able to replicate this.

As a quick fix for now could you toggle off the grid feature. 
1. Click on the `Screen` inside your component tree
2. On the right side of your screen you should see a section called `Layout`
3. Click the first icon
4. This should be using the previous layout before the new grid was implemented. Your table now hopefully be full screen.

I've attached an image below.
<img width="1509" alt="Screenshot 2024-09-02 at 11 32 35" src="https://github.com/user-attachments/assets/5589d668-c2d4-4b3a-be57-1633ecad3b3e">

I hope this helps, thanks in advance for more context.

**Reply by aviNrich** [reply]
Solved by clickin on the Lyaout. I think this should be the default.
Thank you
